src/Microsoft.ServiceFabric.Actors.Tests/Microsoft.ServiceFabric.Actors.Tests.csproj

```diff
@@ -0,0 +1,117 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
```

Comment:
Could you please add a file service_fabric_managed_test.props similar to service_fabric_managed_prod.props and then include that file in these .csprojs. Please refer to existing .csprojs and then change these .csproj in notepad. Also create these as normal class library projects and not as test projects.